Trippy/Utils/ParseUtils.h

```diff
@@ -0,0 +1,27 @@
+//
+//  ParseUtils.h
+//  Trippy
+//
+//  Created by Catherine Lu on 7/7/22.
+//
+
+#import <Foundation/Foundation.h>
+#import "Parse/Parse.h"
+#import "Collection.h"
+#import "Location.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+
+@interface ParseUtils : NSObject
```

Comment:
I see, I did have this isolation in mind - I believe that the methods returning Parse objects are used only by my query handlers, so only those + ParseUtils interact with Parse. Definitely got a bit messy for sure, I'll make sure to about refactor later (probably by subclassing my data models so that ParseUtils is entirely obsolete).